analyzer/file/zip/main.zeek

```diff
@@ -0,0 +1,44 @@
+module ZIP;
+
+export {
+	type File: record {
+		## True if from global directory header, false if from local file header
+		global_: bool;
+		## File ID associated with content analysis of this file. Only available for local
+		## headers where file content has been further processed.
+		fid: string;
```

Comment:
Can we mark this as `&optional` or does this not work?

Reply:
done